features/step_definitions/file_steps.coffee

```diff
@@ -5,17 +5,21 @@ path = require 'path'
 module.exports = ->
 
   @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^I have a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
Here is a competing style preference between you and me. I prefer
```coffee
outputTestFile(
  path.join(@tmpDir, 'example_spec.coffee')
  passing: status is 'passing'
  done)
```
because I'm against right alignment.